front/lib/models.ts

```diff
@@ -1123,6 +1124,11 @@ ExtractedEvent.init(
       allowNull: false,
       defaultValue: DataTypes.NOW,
     },
+    marker: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      defaultValue: "",
```

Comment:
// This is cracra, I know 😬 
Default value for existing events as we already have a few in prod, to avoid having to create a migration file!